owlbot.py

```diff
@@ -38,14 +38,15 @@
 # # Add templated files
 # # ----------------------------------------------------------------------------
 templated_files = common.py_library(
-python.py_samples(skip_readmes=True)
     samples=False,  # set to True only if there are samples
     microgenerator=True,
     cov_level=98,
 
 )
 s.move(templated_files, excludes=[".coveragerc"])  # microgenerator has a good .coveragerc file
 
+python.py_samples(skip_readmes=True)
+
```

Comment:
Here again, we need to have `owlbot.py` checked by the `blacken` and `lint` sessions.

Reply:
The issue is tracked here https://github.com/googleapis/synthtool/issues/1044